composer.json

```diff
@@ -20,7 +20,7 @@
         "barryvdh/laravel-ide-helper": "^3.0",
         "larastan/larastan": "^2.9",
         "orchestra/testbench": "^9.0",
-        "phpstan/phpstan": "^1.10",
+        "phpstan/phpstan": "1.11.1",
```

Comment:
locked until https://github.com/phpstan/phpstan/issues/11624 has been resolved

Reply:
I reverted this after the PHPStan fix.